Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -228,27 +209,27 @@ open class ChartDataSet: ChartBaseDataSet
         var entries = [ChartDataEntry]()
         
         var low = 0
-        var high = _values.count - 1
+        var high = values.endIndex - 1
```

Comment:
just keep `.count` so it's consistent with others

Reply:
`count` and `endIndex` aren't necessarily the same, and `endIndex` is safer (when that's really want we want). When we are able to properly add `Collection` conformance to `ChartDataSet` then it will become possible to call this on a subrange of a `ChartDataSet` where `startIndex` isn't guaranteed to be `0` and `endIndex` isn't guaranteed to be `count`.